UML-Diagram/ReturnSystem.puml

```diff
@@ -0,0 +1,28 @@
+@startuml ReturnSystem
```

Comment:
There is no relation on the Return context and other clases so there is no use for this clase